app/taskk.hoon

```diff
@@ -168,7 +168,7 @@
 ::  :-  %+  welp
 ::        %+  turn 
 ::          (prey subp.state hid)
-::        |=  {o/bone *}
+::        |=  [o=bone *]
 ::        [o car]
 ::  [(watch-dir way) +>.$]
 ++  prep  _`.
```

Comment:
Similarly, you should PR this to @vvisigoth's [Taskk repo](https://github.com/vvisigoth/taskk).